### ISSUE TITLE
Perfomance when wrap bottom bar with Stack.Navigator

### ISSUE THREAD

**Issue by ducpt-bili** [hoaphantn7604/react-native-curved-bottom-bar#56]
The bottom tab is so lagging when i combine Curved Bottom Bar and Stack navigator. The layer is like this:
- Stack.Navigator:
   - Stack.Screen with component is Curved Bottom Bar 
   - Stack.Screen is other screen.

Thank you.

**Comment by arun-saleth** [comment]
Yes i am also facing issue while using nesting navigation.when clicking tab icons so much delay to navigate

**Comment by hoaphantn7604** [comment]
hi @ducpt-bili , @arun-saleth ,
What steps can I follow to replicate the issue you are experiencing?
Have you tried using React.memo and then wrapping the BottomBar?

**Comment by arun-saleth** [comment]
@hoaphantn7604  I tried memo.Its woking perfectly.Thank you for your Suggestions

**Comment by jelyqs** [comment]
Hi @arun-saleth

Can you share with me where to handle useMemo?

**Comment by Sulaiman122** [comment]
@hoaphantn7604 @arun-saleth 
it's slow for me, I don't know why, tried useMemo didn't change
maybe i wrote it wrong, could you explain how to use useMemo or if there is another solution to this issue please help

**Comment by anhnguyen123** [comment]
> CHÀO@ducpt-bili,@arun-saleth, Tôi có thể làm theo những bước nào để tái hiện vấn đề bạn đang gặp phải? Bạn đã thử sử dụng React.memo và sau đó gói BottomBar chưa?

Do you mean wrap usememo to cover the entire bottom bar?
